relay-server/src/processing/spans/mod.rs

```diff
@@ -22,7 +22,7 @@ mod filter;
 mod integrations;
 mod process;
 #[cfg(feature = "processing")]
-mod store;
+pub mod store;
```

Comment:
```suggestion
mod store;
```

Doesn't seem necessary.